### PR TITLE
Added Jersey-based Lambda Handler

### PR DESCRIPTION
**Pull request by przemos**
That is the initial codebase to base further work on. Apart from functionality needed for the sprint BL-2263 it contains changes used in PoC like KMS-encrypted config that will be needed in future sprints.

Restructuring repo to introduce split between webapp and selenium

Added static assets url as system variable

Added Cache-control filter

Added request/response logging: AccessLogger

Added X-Frame-Options: DENY response header

### CODE REVIEW

File: webapp/src/main/resources/template/home.hbs [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{{#partial "content" }}
```
Comment (kevinc-kainos) [inline webapp/src/main/resources/template/home.hbs]: perhaps "home" for the partial name

File: webapp/src/main/resources/template/home.hbs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <h1>
+                        <span class="content-header__type">
+
+                        </span>
+
+            <span class="content-header__title">
+                            Sign up for an MOT reminder
+                        </span>
+
+        </h1>
```
Comment (kevinc-kainos) [inline webapp/src/main/resources/template/home.hbs]: here and also in the master.hbs there's some unnecessary spacing

File: webapp/src/main/resources/template/home.hbs (hunk 1 above)
Comment (przemos) [inline webapp/src/main/resources/template/home.hbs]: "content" is the placeholder defined in master template and designates the place where main content of the page should be defined.

File: webapp/src/main/resources/template/home.hbs (hunk 2 above)
Comment (przemos) [inline webapp/src/main/resources/template/home.hbs]: fixed

**Pull request merged**